openproblems/tasks/spatial_decomposition/methods/nmfreg.py

```diff
@@ -0,0 +1,97 @@
+from ....tools.decorators import method
+from ....tools.normalize import log_cpm
```

Comment:
weird, I would have expected that this unused import would have been catched by precommit... do you have flake8 enabled otherwise? pre-commit seems to pass anyway, not a problem, just owndering